src/platform/esp32/architecture.h

```diff
@@ -101,6 +101,8 @@
 #define HW_VENDOR meshtastic_HardwareModel_STATION_G1
 #elif defined(DR_DEV)
 #define HW_VENDOR meshtastic_HardwareModel_DR_DEV
+#elif defined(HELTEC_HRU_3601)
```

Comment:
The current commit in this PR already uses HELTEC_HRU_3601 tag so it should be good to merge